python/cuml/dask/linear_model/logistic_regression.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2019-2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+from dask_glm.estimators import LogisticRegression as LogisticRegressionGLM
+import cupy as cp
+import dask_cudf
+
+
+class LogisticRegression(BaseEstimator):
+
```

Comment:
Just adding to remember myself to check later, we need a docstring for the class

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/3512/commits/2b5d7fb536b7cf2b99b2bbf735e808913c3fc3f2